integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -0,0 +1,103 @@
+    """
+    help = """
+    Transmit enterprise learner course completion data for the given enterprise customer.
+    """
+    integrated_channel_choices = {
+        'sap': SAPSuccessFactorsEnterpriseCustomerConfiguration,
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--enterprise_customer',
+            dest='enterprise_customer',
+            default=None,
+            metavar='ENTERPRISE_CUSTOMER_UUID',
+            help='Transmit data for only this EnterpriseCustomer. '
+            'Omit this option to transmit to all EnterpriseCustomers with active integrated channels.'
+        )
+        parser.add_argument(
+            '--channel',
+            dest='channel',
+            default=None,
+            metavar='INTEGRATED_CHANNEL',
+            help='Transmit data to this IntegrateChannel. '
+            'Omit this option to transmit to all configured, active integrated channels.',
+            choices=self.integrated_channel_choices.keys(),
+        )
+
+    def handle(self, *args, **options):
+        """
+        Transmit the learner data for the EnterpriseCustomer(s) to the active integration channels.
+        """
+        # If a valid enterprise customer uuid was provided, transmit for that customer.
+        # Otherwise, transmit for all customers.
+        enterprise_customer_uuid = options['enterprise_customer']
+        if enterprise_customer_uuid:
+            try:
+                enterprise_customer = EnterpriseCustomer.active_customers.get(uuid=enterprise_customer_uuid)
+            except EnterpriseCustomer.DoesNotExist:
+                raise CommandError('Enterprise customer {} not found, or not active'.format(enterprise_customer_uuid))
+        else:
+            enterprise_customer = None
+
+        # Assemble a list of integrated channel classes to transmit to.
+        # If a valid channel type was provided, use it.
+        # Otherwise, use all the available channel types.
+        channel_code = options['channel']
+        if channel_code is not None:
+            if channel_code not in self.integrated_channel_choices:
+                raise CommandError('Invalid integrated channel: {}'.format(channel_code))
+            channel_classes = [self.integrated_channel_choices.get(channel_code)]
+        else:
+            channel_classes = self.integrated_channel_choices.values()
+
+        # Loop through each channel class (optionally for a specific enterprise customer)
+        active_integrated_channel_found = False
+        for channel_class in channel_classes:
+            integrated_channels = channel_class.objects.filter(active=True)
+            if enterprise_customer:
+                integrated_channels.filter(enterprise_customer=enterprise_customer)
+
+            # Transmit the learner data to each integrated channel
+            for integrated_channel in integrated_channels:
+                self.transmit_learner_data(integrated_channel)
```

Comment:
I didn't even know that `@celery.task` wasn't asynchonous by default!  Thanks for noticing that.. could have ground jenkins to a halt 😟 

For code clarity, I chose to have the [no djcelery stub code add a`delay` attribute](https://github.com/open-craft/edx-enterprise/blob/479c463f00006c73500c8333b76ceb2a3d97d250/integrated_channels/integrated_channel/management/commands/__init__.py#L11-L17) to the wrapped function, so you don't have to catch the `AttributeError`.  And since it's useful for your PR too, I've moved both the celery import and the stub into [management/commands/__init__.py](https://github.com/edx/edx-enterprise/pull/59/commits/479c463f00006c73500c8333b76ceb2a3d97d250#diff-11c9715fd5316220a126561ca420c9dcR1).

Let me know if you see any issues with this, or if I should move it somewhere else?